Add kubernetes dashboard to infra helmfile (#31)
chainstack/graph-deployment: helmfile/INFRA_README.md

### Let's Encrypt prod clusterissuer
Small staff helm chart for configuring cert-manager to use let's encrypt for issuing certs.

### Kubernetes Dashboard
https://kubernetes.io/docs/tasks/access-application-cluster/web-ui-dashboard/

> This dashboard allows is WebUI allows you to see and edit workloads, read pod logs and etc.

Reviewer: ```suggestion
WebUI allows you to see and edit workloads, read pod's logs and etc.
```

## Installation
Copy `values-example.yaml` to `values.yaml`. Replace values with actual values.

Run following command:
```
helmfile -n <namespace> apply
```

## Usage
### Kubernetes Dashvoard

> Run following command it would return address (for me it's I'll use it in next steps):

Reviewer: ```suggestion
Run the following command to configure a proxy between your local machine and k8s API:
```

```
kubectl proxy
```

> Replace in following url `<namespace>` with actual namespace used during installation and open it in browser `http://127.0.0.1:8001/api/v1/namespaces/<namespace>/services/https:kubernetes-dashboard:https/proxy/`.

Reviewer: ```suggestion
Open the following link in a browser. Don't forget to replace `<namespace>` with the actual namespace used during the installation. 
`http://127.0.0.1:8001/api/v1/namespaces/<namespace>/services/https:kubernetes-dashboard:https/proxy/`.
```

> On login page select `kubeconfig` option. Now you need to upload your kubeconfig file, that is located by default in `~/.kube/config`. Note: typically this directory is hidden.

Reviewer: ```suggestion
On login page, select `kubeconfig` option and upload your kubeconfig file located in `~/.kube/config` by default. Note: typically this directory is hidden.
```
